app/Jobs/LaunchSearchRecipeJob.php

```
@@ -33,21 +29,39 @@
 
-        // TODO: Validate SearchRecipe input before processing
-        // TODO: Map to SearchRecipeDTO
-        // TODO: Dispatch to external search service
-        // TODO: Save search result snapshots (optional)
+        try {
+            $domainModel = SearchRecipeMapper::fromEloquentToDomain($this->searchRecipe);
+            $payload     = SearchRecipeMapper::fromDomainToArray($domainModel);
+            $response    = Http::post(config('services.python_api.search_recipe_crawler_url'), $payload);
+
+            if ($response->failed()) {
+                throw new CrawlerResponseException('External service failed to process SearchRecipe.');
```

PHPStan (2.0.3)

Instantiated class App\Jobs\CrawlerResponseException not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)